CHANGELOG.md

```diff
@@ -1,5 +1,71 @@
 # Release Notes
 
+## Version 1.22.0-beta.0 - 2021-06-28
+
+Features:
+
+* Add auto pause on arm64 image [#11743](https://github.com/kubernetes/minikube/pull/11743)
+* Addon list: Add info on each addon's maintainer  [#11753](https://github.com/kubernetes/minikube/pull/11753)
+* Add ability to pass 'max' value to memory and cpus flags [#11692](https://github.com/kubernetes/minikube/pull/11692)
+
+Bugs:
+
+* Fix --base-image caching for images specified by name:tag [#11603](https://github.com/kubernetes/minikube/pull/11603)
+* Fix embed-certs global config [#11576](https://github.com/kubernetes/minikube/pull/11576)
+* Fix a download link to use arm64 instead of amd64 [#11653](https://github.com/kubernetes/minikube/pull/11653)
+* Move daemon cache check before file cache check [#11690](https://github.com/kubernetes/minikube/pull/11690)
+* Move node config deletion out of drainNode and into Delete [#11731](https://github.com/kubernetes/minikube/pull/11731)
+* gcp-auth: do not override existing environment variables in pods [#11665](https://github.com/kubernetes/minikube/pull/11665)
+
+Minor improvements:
+
+* Allow running amd64 binary on M1 [#11674](https://github.com/kubernetes/minikube/pull/11674)
+* Upgrade containerd config [#11632](https://github.com/kubernetes/minikube/pull/11632)
+* Improve French locale [#11728](https://github.com/kubernetes/minikube/pull/11728)
+* Do not return an error from Systemd.ForceStop(svc) if svc is already stopped [#11667](https://github.com/kubernetes/minikube/pull/11667)
```

Comment:
what does it do for user, change wording how does this fix user's problem 
something like `Fix UI error on contained error spam...`

Reply:
done